terminus-ui/autocomplete/src/autocomplete.component.ts

```diff
@@ -806,22 +815,19 @@ export class TsAutocompleteComponent implements OnInit,
       }
 
       // Add to the collection
-      const newSelection = this.autocompleteSelections.slice();
-      newSelection.push(selection.option.value);
-      this.autocompleteSelections = newSelection;
+      this.autocompleteSelections = this.autocompleteSelections.concat(selection.option);
```

Comment:
`.concat` returns a new array, right? we had to do the way we're doing because engage reducer wouldn't allow mutation...

Reply:
yes'mm it does indeed

Reply:
Yep it does